art/estimators/object_detection/pytorch_yolo.py

```diff
@@ -43,74 +43,78 @@ def translate_predictions_xcycwh_to_x1y1x2y2(
     y_pred_xcycwh: "torch.Tensor", input_height: int, input_width: int
 ) -> List[Dict[str, "torch.Tensor"]]:
     """
-    Convert object detection predictions from xcycwh to x1y1x2y2 format.
+    Convert object detection predictions from xcycwh (YOLO) to x1y1x2y2 (torchvision).
 
-    :param y_pred_xcycwh: Labels in format xcycwh.
-    :return: Labels in format x1y1x2y2.
+    :param y_pred_xcycwh: Object detection labels in format xcycwh (YOLO).
+    :param height: Height of images in pixels.
+    :param width: Width if images in pixels.
```

Comment:
```suggestion
    :param input_height: Height of images in pixels.
    :param input_width: Width if images in pixels.
```

Reply:
I think it actually makes more sense to rename the variable than the docstring. The current names `input_height` and `input_width` are slightly redundant since it should be clear that they refer to the input image.